Propulsion/Oarhole.md

```diff
@@ -0,0 +1,27 @@
+**_oarhole, oarport_** (English); _årehul_ (Danish); _Riemenloch_ (German)
+
+_**hár** f., pl. háir_ (Old Norse) [citation: [prose](https://onp.ku.dk/onp/onp.php?o31447)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=31982)]     
```

Comment:
These links are to the wrong definitions: 
https://lexiconpoeticum.org/m.php?p=lemma&i=31978
https://onp.ku.dk/onp/onp.php?o31447